site.yml

```diff
@@ -18,11 +18,14 @@
   environment: env
   roles: 
   - { role: base }
+  - { role: vagrant }
```

Comment:
`vagrant` role is a pretty nice introduction :+1: 

I can see that we might be able to use it for baremetal environments as well that run behind proxy and for setting up our devtest machines. In future, I am thinking we could enhance this role to be a little more generic for setting up bashrc for any user. Having this role gives me that way! Hope you will be ok with me making enhancements to this role.